mephisto/abstractions/providers/mturk/mturk_utils.py

```diff
@@ -506,6 +506,7 @@ def create_compensation_hit_with_hit_type(
     is_sandbox = client_is_sandbox(client)
 
     # Create the HIT
+    # Question: Does method below need to be updated as well?
```

Comment:
question

Reply:
Good question - not this one. Here we're just putting up a static task for a worker to complete on their own time, expiration here is essentially meaningless.

Would be worthwhile to include the answer in the comment
```
# Create a compensation HIT to be completed in the next month
```